.github/workflows/ci.yml

```diff
@@ -57,3 +57,26 @@ jobs:
       - uses: actions/checkout@v3
       - name: Run installer.sh
         run: sh docs/.vuepress/public/installer.sh
+      - name: Check kumactl architecture
+        run: if ./kuma-*/bin/kumactl version | grep amd64 &> /dev/null; then exit 0; else exit 1; fi
+  installer-sh-arm64:
+    runs-on: ubuntu-latest
+    name: Test installer.sh on arm64
+    steps:
+      - uses: actions/checkout@v3
+      - uses: uraimo/run-on-arch-action@v2
+        name: Run commands
+        id: runcmd
+        with:
+          arch: aarch64
+          distro: ubuntu22.04
+
+          # Not required, but speeds up builds by storing container images in
+          # a GitHub package registry.
+          githubToken: ${{ github.token }}
+
+          # Set an output parameter `uname` for use in subsequent steps
+          run: |
+            apt-get update && apt-get install -y curl tar gzip
+            sh docs/.vuepress/public/installer.sh
+            if ./kuma-*/bin/kumactl version | grep arm64 &> /dev/null; then exit 0; else exit 1; fi
```

Comment:
This is not going to work for future version as the arch shouldn't be in the version to start with.

Reply:
Changed to `file kumactl` and check which architecture it returns.